client/src/leap/soledad/client/adbapi.py

```diff
@@ -204,7 +204,7 @@ def runU1DBQuery(self, meth, *args, **kw):
         :rtype: twisted.internet.defer.Deferred
         """
         meth = "u1db_%s" % meth
-        semaphore = BoundedSemaphore(SQLCIPHER_MAX_RETRIES - 1)
+        semaphore = DeferredSemaphore(SQLCIPHER_MAX_RETRIES - 1)
```

Comment:
it's fixed in #320 (I understand you mean SQLCIPHER_MAX_RETRIES, not SQLCIPHER_MAX_RETRIES -1)